toupee/ensembles/methods.py

```diff
@@ -202,4 +219,78 @@ def _on_model_end(self):
 
     def _on_model_start(self):
         """ Default callback when a model starts training """
-        pass
\ No newline at end of file
+        self.data = self.data.resample()
+
+
+class DynamicMembers(EnsembleMethod):
+    """
+    A simple Ensemble - repeat the training N times and aggregate the results
+    """
```

Comment:
Two minor comments:
- If this one is called `DynamicMembers`, the class in L152 should probably be `StaticMembers` (clearer naming than `Simple`)
- Related to the point above: if the other type of ensembles stay as `Simple`, then this docstring becomes ambiguous

Reply:
I fixed the docstring, but there will be other types of ensemble types, so renaming `Simple` isn't as... simple.